docs/LibraryEvolution.rst

```diff
@@ -903,11 +904,15 @@ not handle new cases well.
 Protocols
 ~~~~~~~~~
 
-There are very few safe changes to make to protocols:
+There are very few safe changes to make to protocols and their members:
 
 - A new non-type requirement may be added to a protocol, as long as it has an
   unconstrained default implementation.
 - A new associated type may be added to a protocol, as long as it has a default.
+  As with any other declarations, newly-added associated types must be marked
+  with ``@available`` specifying when they were introduced.
+- A default may be added to an associated type. Removing a default is a
```

Comment:
I don't see how removing a default can be binary compatible.

Suppose version 2 adds an associated type with a default, and version 3 removes the default. A client compiled against version 1 will not have the associated type in its witness table.